modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -146,6 +146,17 @@ The following permissions are not required to create the private {product-title}
 * `Microsoft.Storage/storageAccounts/write`
 ====
 
+.Required permissions for creating an image registry storage account private endpoint
```

Comment:
@jinyunma Should the required permissions here be updated to include the following? or should we leave it as is. 

```suggestion
      - Microsoft.Storage/storageAccounts/blobServices/read
      - Microsoft.Storage/storageAccounts/blobServices/containers/read
      - Microsoft.Storage/storageAccounts/blobServices/containers/write
      - Microsoft.Storage/storageAccounts/blobServices/generateUserDelegationKey/action
      - Microsoft.Storage/storageAccounts/read
      - Microsoft.Storage/storageAccounts/write
      - Microsoft.Storage/storageAccounts/delete
      - Microsoft.Storage/storageAccounts/listKeys/action
      - Microsoft.Resources/tags/write
      - Microsoft.Network/privateEndpoints/write
      - Microsoft.Network/privateEndpoints/read
      - Microsoft.Network/privateEndpoints/privateDnsZoneGroups/write
      - Microsoft.Network/privateEndpoints/privateDnsZoneGroups/read
      - Microsoft.Network/privateDnsZones/read
      - Microsoft.Network/privateDnsZones/write
      - Microsoft.Network/privateDnsZones/join/action
      - Microsoft.Network/privateDnsZones/A/write
      - Microsoft.Network/privateDnsZones/virtualNetworkLinks/write
      - Microsoft.Network/privateDnsZones/virtualNetworkLinks/read
      - Microsoft.Network/virtualNetworks/subnets/join/action
      - Microsoft.Network/virtualNetworks/join/action
      - Microsoft.Network/networkInterfaces/read
      - Microsoft.Storage/storageAccounts/PrivateEndpointConnectionsApproval/action
  ```

Reply:
I think we should leave it as is now, no need to update.
Because you could find those new permissions(compared with the original permissions list) have already been added in other sections (required permissions for xxxxx) in doc [installing-azure-account#minimum-required-permissions-ipi-azure_installing-azure-account](https://69087--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-account#minimum-required-permissions-ipi-azure_installing-azure-account)) , so no need to duplicate them here.